code/demo_player/editor/ui/AttachmentEditorMenu.razor

```diff
@@ -1,5 +1,5 @@
 ﻿@using Sandbox.UI;
-@attribute [StyleSheet("/swb_base/editor/ui/AttachmentEditorMenu.scss")]
+@attribute [StyleSheet("/demo_player/editor/ui/AttachmentEditorMenu.scss")]
```

Comment:
I was thinking `swb_player` might be a better name since most consumers will use the player and base together.

Reply:
I will rename the folder